<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyPage</class>
 <widget class="QDialog" name="MyPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="captionNichname">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Nickname</string>
   </property>
  </widget>
  <widget class="QLabel" name="captionRating">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Rating</string>
   </property>
  </widget>
  <widget class="QLabel" name="captionResults">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Game Results</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editNickName">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelRating">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>1000</string>
   </property>
  </widget>
  <widget class="QLabel" name="captionDraws">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Draws</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelResults">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>181</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>7 wins, 8 losses</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDraws">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="captionFouls">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Fouls</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFouls">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>O K</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>MyPage</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>392</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
